guides/v2.3/comp-mgr/cli/cli-upgrade.md

```diff
@@ -10,128 +10,105 @@ functional_areas:
 {% capture ee %}{{site.data.var.ee}}{% endcapture %}
 -->
 
-Upgrade Magento from the command line if you installed the software using any of the following:
+You can upgrade Magento from the command line if you installed the software by:
 
-* Downloaded the {% glossarytooltip 7490850a-0654-4ce1-83ff-d88c1d7d07fa %}metapackage{% endglossarytooltip %} using `composer create-project`
-* Installed the compressed archive
+* Downloading the {% glossarytooltip 7490850a-0654-4ce1-83ff-d88c1d7d07fa %}metapackage{% endglossarytooltip %} using `composer create-project`.
+* Installing the compressed archive.
 
-There are two ways to upgrade your Magento application to 2.3:
-1. Manual: [Upgrade using the command line].
-2. Semi-automated: [Upgrade using the script].
-The upgrading scenario is the same from the system's point of view except that several steps are automated using the PHP script.
+There are two ways to upgrade your Magento application to the 2.3 version:
+1. [Upgrade using the command line] (manual process)
+2. [Upgrade using the script] (semi-automated process)
+
+The upgrade scenario is the same for each of these options (both utilize Composer and a command line interface) from the system's point of view. However, if you upgrade using the script several of the steps are automated.
 
 {:.bs-callout .bs-callout-warning}
-Both ways require that you comply with the pre-upgrade checklist.
+Both scenarios require that you comply with the [Pre-upgrade checklist].
 
 {:.bs-callout .bs-callout-info}
-If you cloned the Magento 2 GitHub repository, you **cannot** use this method to upgrade. Instead, see [Update the Magento application].
-
-## Pre-upgrade checklist
-
-Comply with the the [Update and upgrade checklist].
-
-### Prerequisite for the `pub` directory root {#upgrade-cli-pub}
+If you cloned the Magento 2 GitHub repository, you **cannot** use this method to upgrade. Instead, see [Update the Magento application] for upgrade instructions.
 
-This section applies to you *only* if you set the Magento root directory to `<your Magento install dir>/pub`.
-If you did not do this, skip this section and continue with the next section.
+## Prerequisites
 
-For the upgrade, create another subdomain or docroot that uses the Magento installation directory as its root.
-Run the [System Upgrade utility] using that subdomain.
+Complete the following prerequisites to prepare your environment before starting the upgrade process:
+* **Complete Update and upgrade checklist**—To avoid possible errors during installation or upgrading, complete the [Update and upgrade checklist].
```

Comment:
Just a nit pick on caps: Complete the update and upgrade checklist

Reply:
The title of the page and checklist is "Update and upgrade checklist" -- still no caps?

Reply:
Ah I see, maybe Complete shouldn't be bolded? The name Update being capped is fine.

Reply:
Yeah, each of these three bullet points have bolded intro text to denote the step that needs to be taken, it's not just about the page name being bolded. I'll think about this and see if it should be communicated a little differently.

Reply:
![screen shot 2018-09-13 at 9 48 14 am](https://user-images.githubusercontent.com/14031342/45496039-2fa30880-b73a-11e8-92ac-5b0fc3cf17d4.png)